TutorialApplication/MainViewController.m

```diff
@@ -11,12 +11,16 @@
 #import "Organization.h"
 #import "DetailViewController.h"
 #import "CreateEmployeeViewController.h"
+#import "DatabaseController.h"
+#import "AppDelegate.h"
 
 @interface MainViewController () <CreateEmployeeDelegate>
 
-@property (strong, nonatomic) Organization *organization;
+@property (strong, nonatomic) FFOrganization *organization;
 
-@property (weak, nonatomic) Employee *selectedEmployee;
+@property (strong, nonatomic) NSArray<FFEmployee*> *myEmployees;
```

Comment:
@property (strong, nonatomic) NSArray<FFEmployee*> *myEmployees;

to

@property (strong, nonatomic) NSArray<FFEmployee *> *myEmployees;